src/components/Section/Foreign/Foreign.test.jsx

```diff
@@ -3,219 +3,23 @@ import configureMockStore from 'redux-mock-store'
-        section: 'business/family',
-        action: component => {
-          component
-            .find('.foreign-business-family .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/employment',
-        action: component => {
-          component
-            .find('.foreign-business-employment .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/ventures',
-        action: component => {
-          component
-            .find('.foreign-business-ventures .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/conferences',
-        action: component => {
-          component
-            .find('.foreign-business-conferences .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/contact',
-        action: component => {
-          component
-            .find('.foreign-business-contact .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/sponsorship',
-        action: component => {
-          component
-            .find('.foreign-business-sponsorship .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/political',
-        action: component => {
-          component
-            .find('.foreign-business-political .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'business/voting',
-        action: component => {
-          component
-            .find('.foreign-business-voting .branch .yes input')
-            .simulate('change')
-        }
-      },
-      {
-        section: 'travel',
-        action: component => {
-          component
-            .find('.foreign-travel .branch .yes input')
-            .at(0)
-            .simulate('change')
-        }
-      }
-    ]
-
-    tests.forEach(test => {
-      const component = mount(
-        <Provider store={store}>
-          <MemoryRouter>
-            <Foreign section="foreign" subsection={test.section} />
-          </MemoryRouter>
-        </Provider>
-      )
-      test.action(component)
-      expect(component.find('div').length).toBeGreaterThan(0)
-    })
-  })
-
-  it('can parse previous names', () => {
-    const store = mockStore({
-      application: {
-        Identification: {
-          ApplicantName: {
-            first: 'john',
-            last: 'smith'
-          },
-          OtherNames: {
-            HasOtherNames: {
-              value: 'Yes'
-            },
-            List: {
-              items: [
-                {
-                  Item: {
-                    first: 'johnathan',
-                    last: 'smith'
-                  }
-                }
-              ]
-            }
-          }
-        }
-      }
     })
-    const component = mount(
```

Comment:
I deleted these tests because they aren't effective. The test is expecting a `.div`. This is always going to pass because there are `div`'s all over the section.  It doesn't necessarily mean it's going to the correct section.

We can reintroduce tests, but I don't know how necessary it is to test React Router functionality.

Reply:
It looks like they were also looking for specific elements on the page (and simulating `change` events on them).. I totally agree that we should not be testing React Router functionality, and that these tests were far from ideal, I just wonder about explicitly deleting them at this point, assuming they should still be passing. I don't think it's a big deal one way or the other though 🤷‍♀️